docs/reference/provisioning-methods.rst

```diff
@@ -0,0 +1,120 @@
+Supported provisioning methods
+===================================
+
+This document lists the basic provisioning methods currently supported by Testflinger device connectors. 
+
+Provision data are specified as a key-value pair in the ``provisioning_data`` section of a Testflinger job definition file:
+
+.. code-block:: yaml
+
+  job_queue: my-queue-name
+  provision_data:
+    key: value    # Data to pass to the provisioning step
+
+The type of provision data usually varies on device types. For example, our MAAS provisioner currently only needs to to know which ``distro`` series to use, whereas the devices that install all-snaps images (such as Dragonboard) may need to know which channel to use when building the image instead.
```

Comment:
The final sentence does not match the device_connector or the table below - Dragonboard takes a URL not a channel

Reply:
fixed